lib/tasks/jade.js

```diff
@@ -0,0 +1,31 @@
+var gulp = require('gulp');
+var notify = require("gulp-notify");
+var config = require('../config').jade;
+var jade = require('gulp-jade');
+var newer = require('gulp-newer');
+
+function compileJade(onlyNewFiles) {
+    var options = {
+        pretty: config.hasOwnProperty("indentation") ? config.indentation : false
+    };
+
+    var stream = gulp.src(config.src)
+        .pipe(jade(options));
+
+    if (onlyNewFiles) {
+        stream = stream.pipe(newer({ dest: config.dest }));
+    }
+
+    stream = stream.pipe(gulp.dest(config.dest))
+        .pipe(notify('JADE compiled: <%= file.relative %>'));
+
+    return stream;
+}
+
+gulp.task('jade', function() { return compileJade(); });
+
+gulp.task('jade:watch', function() {
+  var filesToWatch = config.hasOwnProperty("watch") ? config.watch : config.src;
+  gulp.watch(filesToWatch, {'dot': true}, function() { compileJade(true); });
```

Comment:
I can't tell, this `jade:watch` block was duplicated from another task you have previously created. Just wanted to make sure it worked and eventually forgot to check that.

Either way, please don't merge this just yet. I managed to create some kind of infinite loop or memory leak of some sort that completely froze my computer yesterday. I'll go over this once again and see what I missed.